deploy/testnet/revert.sh

```
@@ -16,7 +16,7 @@ source ./config.env
 
 container_name="honest_worker"
 chainspec="./chainspec.json"
-rpc_api_endpoint="http://127.0.0.1:9944"
+#rpc_api_endpoint="http://127.0.0.1:9944"
```

It looks like this variable is being used here. I remember there was an unused variable error and it seems strange.

---

Yes, it sends a warning that it is not being used.
```
In deploy/testnet/revert.sh line 19:
rpc_api_endpoint="http://127.0.0.1:9944"
^--------------^ SC2034 (warning): rpc_api_endpoint appears unused. Verify use (or export if used externally).
```
Maybe this line should be uncommented? And add a command, for example `echo "$rpc_api_endpoint"`. This command will be used as a stub for the linter and will not trigger the warning

---

Just ensure this script do not call another one where this variable is expected to be set, and then remove it entirely. Also you can add `set -u` to the scripts to fail any attempts to use undefined variables.

---

Done